src/utilities/__tests__/extendSchema-test.js

```diff
@@ -90,6 +90,7 @@ const testSchema = new GraphQLSchema({
     }),
   }),
   types: [FooType, BarType],
+  allowedLegacyNames: ['__badName'],
```

Comment:
I don't think it worth to add this option to common test setup
It's better to define separate minimal schema inside test-case itself.

Reply:
I like that, I just figured that this file was adding all fixture data to this single schema and followed suit. Happy to revisit and create a tiny schema in the test, though 👍